### ISSUE TITLE
User the Importtttttttttttttttttttttttttttttt please, My project is having issues because I'm using 'import' instead of 'const require.' Please update that

### ISSUE THREAD

**Issue by ManoelMorais** [scottie1984/swagger-ui-express#362]
import swaggerDocument from "../swagger.json" assert { type: "json"};
                                                                             ^^^^^^
Nov 1 11:02:35 AM  
Nov 1 11:02:35 AM  SyntaxError: Unexpected identifier
Nov 1 11:02:35 AM      at Loader.moduleStrategy (internal/modules/esm/translators.js:145:18)
Nov 1 11:02:35 AM  npm ERR! code ELIFECYCLE
Nov 1 11:02:35 AM  npm ERR! errno 1
Nov 1 11:02:35 AM  npm ERR! one@1.0.0 start: `node index`
Nov 1 11:02:35 AM  npm ERR! Exit status 1

**Comment by jacobhumston** [comment]
ESM doesn't support JSON. 

You're going to need to do something like this with the file system module;
```js
import { readFileSync } from 'node:fs';

const swaggerDocument = JSON.parse(readFileSync('swagger.json').toString('utf-8'))
```

*You should also consider titling issues you create more professionally...*

**Comment by paulmillr** [comment]
> ESM doesn't support JSON.

it does, use import `with { type: 'json' }` on most recent nodejs

**Comment by jacobhumston** [comment]
> > ESM doesn't support JSON.
> 
> it does, use import `with { type: 'json' }` on most recent nodejs

I don't think that was the case at the time I replied (6ish months ago) if I'm not mistaken.

I've marked my reply as outdated, appreciate it!

**Comment by Rajkumar7633** [comment]
Same issue in import apiDocs from './swagger.json' assert {type: 'json'};
                                     ^^^^^^

SyntaxError: Unexpected identifier 'assert'
    at compileSourceTextModule (node:internal/modules/esm/utils:337:16)
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:163:18)
    at callTranslator (node:internal/modules/esm/loader:430:14)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:436:30)
    at async ModuleJob._link (node:internal/modules/esm/module_job:106:19)

Node.js v22.2.0

**Comment by rajrasane04** [comment]
> Same issue in import apiDocs from './swagger.json' assert {type: 'json'};
>                                      ^^^^^^
> 
> SyntaxError: Unexpected identifier 'assert'
>     at compileSourceTextModule (node:internal/modules/esm/utils:337:16)
>     at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:163:18)
>     at callTranslator (node:internal/modules/esm/loader:430:14)
>     at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:436:30)
>     at async ModuleJob._link (node:internal/modules/esm/module_job:106:19)
> 
> Node.js v22.2.0

Did you find out what it was , cause now I am having that error , says assert is unexpected identifier, I did everything, also updated my node to latest , but still not working •́⁠ ⁠ ⁠‿⁠ ⁠,⁠•̀
